lib/mayaUsd/fileio/jobs/writeJob.cpp

```diff
@@ -163,10 +163,74 @@ bool UsdMaya_WriteJob::Write(const std::string& fileName, bool append)
     return true;
 }
 
+static MStringArray GetExportDefaultPrimCandidates(const UsdMayaJobExportArgs& exportArgs)
```

Comment:
This is the core of the change: select a default prim from the available Maya nodes. This reuse the existing code used to show the selection of possible default prims in the export UI. We need to reuse it in the case the export was done by a direct command and no default prim were given and no request for having no default prim was made (by using specifying that the default prim be "None").